src/styles/layoutMixins.ts

```diff
@@ -44,6 +44,7 @@ const scrollAreaFadeStart = css`
 
   /* Rules */
   position: relative;
+  overflow: hidden;
```

Comment:
gotta imagine this is going to cause pain SOMEWHERE, but I've got no clue where

Reply:
oh rip good catch - it's preventing the "More" dropdown from showing. Man lemme fix